hal-base/hal/constants.py

```diff
@@ -1,3 +1,4 @@
+import enum
```

Comment:
Not sure if we wanna use enums for this class. It doesn't *hurt*, but I'm not sure if it helps now or will later